### ISSUE TITLE
rgi installation in Centos 7

### ISSUE THREAD

**Issue by raphenya** [bioconda/bioconda-recipes#11794]
@mjram0s @apetkau

Please see below:

```
I'm sending this email to you just to inform you (perhaps you already know) that in order to install last version of rgi in a completely updated centos 7 using conda environments (with anaconda3 for python 3.X) I had to add a another chanel: conda-forge so the command would be:

conda install -c conda-forge -c bioconda rgi=4.2.2

if I don' t do that the packages filetype and pyahocorasick are not found so rgi can not be installed.

If you just run the comand

conda install -c bioconda rgi

Everything apparently works, but you get version 3.2.1 installed, not the last one.


I actually not installed it with this single command, I had communication problems during the installation and corrupted packages so I end up executing all of this:

conda install -c conda-forge filetype

conda install -c conda-forge pyahocorasick

conda install -c conda-forge -c bioconda rgi=4.2.2

here there was a problem with qt, I had to delete the download package and install it again

conda install -c conda-forge -c bioconda qt

conda install -c conda-forge -c bioconda rgi=4.2.2

then there was a problem with glib (I lost these commands so I don't know the version but it was the same procedure as before, install glib and then again rgi=4.2.2 and this time everything worked)

I hope this helps you in case anybody else find the same problem.

thank you very much

Oscar C.S.

```

**Comment by apetkau** [comment]
Thanks @raphenya. I do believe it is part of the bioconda instructions to include the channel `conda-forge` in addition to `bioconda`, so I think that first issue is expected (though not quite sure why it defaults to 3.2.1 even if you specify 4.2.2).

For the other install issues, we'll look into seeing if we can reproduce them.

**Comment by apetkau** [comment]
So, we've tested installing with anaconda3 on CentOS 7, but we could not reproduce the issue (everything installs fine for us). I suspect the issue above was all caused by a communication problem, leaving their conda environment in an inconsistent state.

By the way, I noticed that the [RGI docs](https://github.com/arpcard/rgi) list Python 3.6 as a dependency, but we've included Python 3.5 and 3.6 in the bioconda package. Everything appears to work for me in 3.5, but if something is going to fail without 3.6 let me know and we can remove support.
